modules/javafx.graphics/src/main/java/com/sun/glass/ui/Application.java

```diff
@@ -768,4 +753,37 @@ public final Optional<Boolean> isKeyLocked(int keyCode) {
                 return Optional.empty();
         }
     }
+
+    /**
+     * Returns the current set of platform properties as a map of platform-specific keys to
+     * arbitrary values. Callers should assume that the returned map is immutable.
+     *
+     * @implSpec Implementations should either return an immutable map, or give up ownership
+     *           of the returned map.
+     * @return the current set of platform preferences
+     */
+    public Map<String, Object> getPlatformPreferences() {
+        return Map.of();
+    }
+
+    /**
+     * Returns a map of platform-specific preference keys to well-known keys.
+     * <p>
+     * For example, the platform-specific key "Windows.UIColor.Foreground" is mapped to the
+     * well-known key "foregroundColor", which makes it easier to write shared code without
+     * depending on platform-specific details.
+     * <p>
+     * The following well-known keys are currently supported, which correspond to the names of color
+     * properties on the {@link com.sun.javafx.application.preferences.PreferenceProperties} class:
+     * <ul>
+     *     <li>foregroundColor
+     *     <li>backgroundColor
+     *     <li>accentColor
+     * </ul>
+     *
+     * @return a map of platform-specific keys to well-known keys
+     */
+    public Map<String, String> getWellKnownPlatformPreferenceKeys() {
```

Comment:
Not really liking the name.  Aren't these keys that are mapped to FX keys?  Perhaps `getPlatformSpecificMappings`

Reply:
I chose `getPlatformKeyMappings` to put more emphasis on the fact that this is a mapping of keys to keys.